src/transport_udp.cpp

```diff
@@ -295,19 +295,19 @@ UDPTransport::fd_reader()
         memcpy(&(r.addr), &remoteAddr, remoteAddrLen);
 
         ++last_context_id;
-        ++last_media_stream_id;
+        ++last_stream_id;
 
         remote_contexts[last_context_id] = r;
         remote_addrs[ra_key] = {
           last_context_id,
-          last_media_stream_id,
+            last_stream_id,
         };
 
         cd.contextId = last_context_id;
-        cd.mStreamId = last_media_stream_id;
+        cd.streamId = last_stream_id;
 
         // Create dequeue
-        dequeue_data_map[last_context_id][last_media_stream_id].setLimit(50000);
+        dequeue_data_map[last_context_id][last_stream_id].setLimit(50000);
```

Comment:
Should 50000 be a named constexpr at the top of this file or in the header somewhere?

Reply:
It should be a config variable, which we should add after this PR. There are several other config variables to add, which requires passing them upstream to the app so that they can set them.